### PR TITLE
Authenticate login

### PR DESCRIPTION
**Pull request by jleejl**
These are working; login, create account, and forgot your password.

These are not working; menu buttons.

Still need to work on ForgotPassword class

### CODE REVIEW

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -12,5 +12,20 @@
...
     <string name="username">Username</string>
     <string name="textpassword">Password</string>
-    <string name="logout">LogOut</string>
+    <string name="logout">Logout</string>
+    <string name="app_logo_name">VRE</string>
+    <string name="app_title_name">Voice Recognition Emergency</string>
+    <string name="username_text">Email or Username</string>
+    <string name="password_text">Password</string>
+    <string name="forgot_password_text">Forgot your password?</string>
+    <string name="or_text">Or</string>
```
Comment (kouYang2015) [inline app/src/main/res/values/strings.xml]:
```suggestion

```
Remove this and the one in the xml. Maybe we don't need to have an Or text

File: app/src/main/res/layout/activity_login.xml [hunk 2]
```diff
@@ -0,0 +1,154 @@
...
+        android:layout_height="wrap_content"
+        android:background="@drawable/round_style"
+        android:text="@string/login"
+        android:textAllCaps="false"
+        android:textSize="18sp"
+        app:layout_constraintLeft_toLeftOf="parent"
+        app:layout_constraintRight_toRightOf="parent"
+        app:layout_constraintTop_toBottomOf="@id/forgot_password" />
+
+    <TextView
```
Comment (kouYang2015) [inline app/src/main/res/layout/activity_login.xml]: Might not need. Remove whole element.
```suggestion

```

**Review by kouYang2015 — APPROVED** [review]
Looks good after those changes.

File: app/src/main/res/values/strings.xml (hunk 1 above)
Comment (jleejl) [reply app/src/main/res/values/strings.xml]: Deleted the string name

File: app/src/main/res/layout/activity_login.xml (hunk 2 above)
Comment (jleejl) [reply app/src/main/res/layout/activity_login.xml]: Deleted the TextView for "or"

File: app/src/main/res/values/strings.xml (hunk 1 above)
Comment (kouYang2015) [reply app/src/main/res/values/strings.xml]: sounds good. You can go ahead and resolve conversation to close this comment.

**Review by GameyMonster — COMMENTED** [review]
Looks good, just modify some code later

**Pull request merged**